CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/player/MCPlayer.java

```diff
@@ -154,7 +154,7 @@ public void copyToClipboard(String value) {
     
     @Override
     public boolean equals(Object other) {
-        return other.getClass() == this.getClass() && ((MCPlayer) other).player == player;
+        return other != null && other.getClass() == this.getClass() && ((MCPlayer) other).player == player;
```

Comment:
```suggestion
        return other != null && other.getClass() == this.getClass() && player != null && player.equals(((MCPlayer) other).player);
```

While you're at it, could you use .equals for the comparison (or, alternatively, Objects.equals if you don't like the null-check ^^)